### PR TITLE
Fix issue #4

### PR DESCRIPTION
**Pull request by skial**
Turned out no macros were involved.

### CODE REVIEW

**Comment by back2dos** [comment]
Thanks, I'll have a look at this, as soon as I fix the tests. I'm a little
worried the resulting behavior for non-optional ints may be quirky, but
we'll see ;)

File: src/tink/web/Stringly.hx [hunk 1]
```diff
@@ -2,6 +2,14 @@ package tink.web;
 
 abstract Stringly(String) to String from String { 
   
+	@:to public function toUnsafeInt():Null<Int> {
+		return Std.parseInt(this);
+	}
+	
+	@:to public function toUnsafeFlot():Null<Float> {
+		return Std.parseFloat(this);
```
Comment (kevinresol) [inline src/tink/web/Stringly.hx]: `parseFloat` doesn't return `null`, but `NaN`, in case of a parse error.

Comment (skial) [inline src/tink/web/Stringly.hx]: Thanks @kevinresol, now updated.

**Comment by back2dos** [comment]
See #4

**Pull request closed without merging**